### ISSUE TITLE
TalkBack is unable to read MarkdownText as part of a merged descendant group

### ISSUE THREAD

**Issue by jstubing** [jeziellago/compose-markdown#144]
I'd like to use the semantics modifier to merge descendants within a container. For example:

```kotlin
Column(
    modifier = Modifier
        .fillMaxWidth()
        .semantics(mergeDescendants = true) {}
) {
    Text("regular text")
    MarkdownText("markdown text")
}
```

With this code I'd expect TalkBack to read "regular text markdown text" as one single readout since the column's descendants have been merged.

Instead, it just reads "regular text". The `MarkdownText` composable remains separate and TalkBack highlights and reads it separately.

Is there any way to fix this such that the `MarkdownText` gets read together with the regular `Text`?
